sdk/core/Azure.Core.Experimental/tests/LowLevelClientTests.cs

```diff
@@ -211,6 +211,79 @@ await client.GetPetAsync("snoopy", new RequestOptions()
             });
         }
 
+        [Test]
+        public async Task CanGetResponseFromLlcLroMethodAsync()
+        {
+            var mockResponse = new MockResponse(200);
+
+            Pet pet = new Pet("snoopy", "beagle");
+            mockResponse.SetContent(SerializationHelpers.Serialize(pet, SerializePet));
+
+            var mockTransport = new MockTransport(mockResponse);
+            PetStoreClient client = CreateClient(mockTransport);
+
+            var data = new
+            {
+                name = "snoopy",
+                species = "beagle"
+            };
+            Operation<BinaryData> operation = await client.ImportPetAsync(RequestContent.Create(data), new RequestOptions());
+            var response = operation.GetRawResponse();
+            var doc = JsonDocument.Parse(response.Content.ToMemory());
+
+            Assert.AreEqual(200, response.Status);
+            Assert.AreEqual("snoopy", doc.RootElement.GetProperty("name").GetString());
+            Assert.AreEqual("beagle", doc.RootElement.GetProperty("species").GetString());
+        }
+
+        [Test]
+        public async Task CanGetOutputModelForLroOnSuccessCodeAsync()
+        {
+            var mockResponse = new MockResponse(200);
+
+            Pet petResponse = new Pet("snoopy", "beagle");
+            mockResponse.SetContent(SerializationHelpers.Serialize(petResponse, SerializePet));
+
+            var mockTransport = new MockTransport(mockResponse);
+            PetStoreClient client = CreateClient(mockTransport);
+
+            var data = new
+            {
+                name = "snoopy",
+                species = "beagle"
+            };
+            Operation<BinaryData> operation = await client.ImportPetAsync(RequestContent.Create(data), new RequestOptions());
+
+            Pet pet = operation.GetRawResponse();
+
+            Assert.AreEqual("snoopy", pet.Name);
+            Assert.AreEqual("beagle", pet.Species);
+        }
+
+        [Test]
+        public async Task CanGetOperationOfOutputModelForLroAsync()
+        {
+            var mockResponse = new MockResponse(200);
+
+            Pet petResponse = new Pet("snoopy", "beagle");
+            mockResponse.SetContent(SerializationHelpers.Serialize(petResponse, SerializePet));
+
+            var mockTransport = new MockTransport(mockResponse);
+            PetStoreClient client = CreateClient(mockTransport);
+
+            var data = new
+            {
+                name = "snoopy",
+                species = "beagle"
+            };
+            LowLevelFuncOperation<Pet> operation = (LowLevelFuncOperation<BinaryData>)await client.ImportPetAsync(RequestContent.Create(data), new RequestOptions());
```

Comment:
I don't think we'll be able to cast directly from `Operation<BinaryData>` to `Operation<OutModel>`, at least not easily.

Instead of doing this cast, is it possible to add a grow-up method to the client (for example, as we do in [Synapse in the convenience layer](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.AccessControl/src/RoleAssignmentsClient.cs#L45)) that creates a new instance of `Operation<OutModel>` (or its LLC subtype) from the `Operation<BinaryData>` returned from the LLC-generated method?